### DISCUSSION TITLE
How many games did they play?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#473] [Galaxy Trivia] [answered]
Susan and Lisa decided to play tennis against each other. They bet $1 on each game they played. Susan won three bets and Lisa won $5.

**Comment by peterbaccaro** [comment, marked as answer]
**Eleven**. Because Lisa lost three games to Susan, she had lost $3 ($1 per game). So, she had to win back that $3 with three more games, then win another five games to win $5.
